tests/translator/output/error_api_mtls_configuration_invalid_type.json

```diff
@@ -1,8 +1,3 @@
 {
-  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [['TruststoreUri', 'TruststoreVersion']] is invalid. MutualTlsAuthentication must be a map with at least one of the following fields ['TruststoreUri', 'TruststoreVersion'].",
```

Comment:
note: The existing error message makes no sense... It is funny to see "Resource with id [['TruststoreUri', 'TruststoreVersion']] is invalid"